### PR TITLE
Add PubSub Publish wrapper

### PR DESCRIPTION
**Pull request by bonedaddy**
Still unsure whether we will allow pubsub subscriptions via this package, so that's why it's not part of this PR.

### CODE REVIEW

File: pubsub.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package rtfs
+
+import (
+	"errors"
+)
+
+// PubSubPublish is used to publish a a message to the given topic
+func (im *IpfsManager) PubSubPublish(topic string, data string) error {
```
Comment (bobheadxi) [inline pubsub.go]: i think this should probs go in `rtfs.go` with the rest of the manager functions (above `CustomRequest`)

Comment (bonedaddy) [reply pubsub.go]: Good point! I guess if we ever add in more advanced PubSub functionality it would be worth having a separate file?

**Review by bobheadxi — APPROVED** [review]

File: pubsub.go (hunk 1 above)
Comment (bobheadxi) [reply pubsub.go]: the issue is that the interface generator wont pick it up

Comment (bobheadxi) [reply pubsub.go]: you could add a new `PubSubManager` class if we have more advanced functionality in the future

**Pull request merged**